### PR TITLE
Reorg: move files from root to NETCORE directory

### PR DESCRIPTION
**Pull request by TimothyMothra**
https://github.com/microsoft/ApplicationInsights-dotnet/issues/1214

Step 1. Reorganize individual repos. This is to ensure that there are no conflicting directories or file names and to guarantee no conflicts when migrating source. This will require some minor changes to existing build definitions.

**Changes**

Update Directory
- EnlistmentRoot stays in root directory
- Add SourceRoot

### CODE REVIEW

File: .vs/ApplicationInsights.AspNetCore/config/applicationhost.config [hunk 1]
```diff
@@ -0,0 +1,990 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cijothomas) [inline .vs/ApplicationInsights.AspNetCore/config/applicationhost.config]: Whats the content of .vs folder?

**Review by cijothomas — CHANGES_REQUESTED** [review]
do we need the .vs folder? 
That aside, the only change is move entire repo inside NETCORE folder, right?

File: .vs/ApplicationInsights.AspNetCore/config/applicationhost.config (hunk 1 above)
Comment (TimothyMothra) [reply .vs/ApplicationInsights.AspNetCore/config/applicationhost.config]: oh that's my mistake. the .vs folder was previously in the gitignore, but it doesn't match against the new rule. i'll remove it now. thanks

**Review by cijothomas — APPROVED** [review]
perhaps a good time to delete nuget.exe. We just need one in the repo. (checked in binaries is bad practice anyways)

**Comment by TimothyMothra** [comment]
Let's delete the binaries after the migration as part of the consolidated step.
When I start authoring new build definitions I'll confirm it's not in use.

**Pull request merged**